### PR TITLE
add pynamical recipe

### PR DESCRIPTION
**Pull request by gboeing**

### CODE REVIEW

File: recipes/pynamical/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    - numba
+
+test:
+  imports:
+    - pynamical
+
+about:
+  home: https://github.com/gboeing/pynamical
+  license: MIT
+  license_family: MIT
```
Comment (jakirkham) [inline recipes/pynamical/meta.yaml]: Please add a `MANIFEST.in` to your project and please add `include LICENSE.txt` to the `MANIFEST.in`.

**Comment by gboeing** [comment]
Any idea why the CI tests are taking 2+ days to run?

**Comment by jakirkham** [comment]
CircleCI was affected by some webhook outage that occurred on GitHub around the same time S3 had issues. Have restarted the CircleCI build.

**Comment by gboeing** [comment]
Thanks @jakirkham

File: recipes/pynamical/meta.yaml (hunk 1 above)
Comment (jakirkham) [inline recipes/pynamical/meta.yaml]: Included below.

**Pull request merged**